R/dynamic.R

```diff
@@ -74,17 +74,18 @@ jekyll = function(
   build_all = function() knit_maybe(input, output, script, method = 'jekyll')
 
   in_dir(dir, {
-    if (!serve) build_all()
-    jekyll_build()
```

Comment:
I was being intentionally conservative here, because `knit_maybe()` only tells me if .Rmd files need to be recompiled. It is a little painful if you update other files, such as CSS. As long as you do not touch other files very often, this PR makes perfect sense.